fagiClient/src/main/java/com/fagi/threads/ThreadPool.java

```diff
@@ -0,0 +1,28 @@
+package com.fagi.threads;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ThreadPool {
+    private static final Object lock = new Object();
+    private List<Thread> threads = new ArrayList<>();
```

Comment:
We should delete not running threads such that it won't make our client take to much memory